### PR TITLE
Sagas/record cluster plot tree test

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

File: src/state/form/sagas/_tests/data/mocks/node.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import moment from 'moment';
+
+import getCurrentUuid from '../utils/getCurrentUuid';
+
+import mockSurvey from './survey';
+
+const node = ({index, parentNode, nodeDefUuid, value}) => ({
```
Comment (SteRiccio) [inline src/state/form/sagas/_tests/data/mocks/node.js:7]: Isn't it better to use the Node factory here?

File: src/state/form/sagas/_tests/data/mocks/surveyDefinitions/cluster/nodeDefs.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+const nodeDefs = {
+  CLUSTER_UUID: {
+    id: 'CLUSTER_ID',
+    uuid: 'CLUSTER_UUID',
+    parentUuid: null,
+    type: 'entity',
+    props: {
+      name: 'cluster',
+      multile: false,
```
Comment (SteRiccio) [inline src/state/form/sagas/_tests/data/mocks/surveyDefinitions/cluster/nodeDefs.js:9]: Typo here...

**Review by SteRiccio — CHANGES_REQUESTED** [review]

File: src/state/form/sagas/_tests/data/mocks/surveyDefinitions/tree/nodeDefs.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+const treeNodeDefs = {
+  TREE_UUID: {
+    id: 'TREE_ID',
+    uuid: 'TREE_UUID',
+    parentUuid: 'PLOT_UUID',
+    type: 'entity',
+    props: {
+      name: 'tree',
+      multile: true,
```
Comment (ramirobg94) [inline src/state/form/sagas/_tests/data/mocks/surveyDefinitions/tree/nodeDefs.js]:
```suggestion
      multiple: true,
```

File: src/state/form/sagas/_tests/data/mocks/surveyDefinitions/tree/nodeDefs.js [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  TREE_KEY_UUID: {
+    id: 'TREE_KEY_ID',
+    uuid: 'TREE_KEY_UUID',
+    parentUuid: 'TREE_UUID',
+    type: 'integer',
+    props: {
+      key: true,
+      name: 'tree_key',
+      multile: false,
```
Comment (ramirobg94) [inline src/state/form/sagas/_tests/data/mocks/surveyDefinitions/tree/nodeDefs.js]:
```suggestion
      multiple: false,
```

File: src/state/form/sagas/_tests/data/mocks/node.js (hunk 1 above)
Comment (ramirobg94) [reply src/state/form/sagas/_tests/data/mocks/node.js:7]: maybe but at the also test that we dont change that, so could be okay

**Review by SteRiccio — APPROVED** [review]

**Pull request merged**